src/main/java/com/github/jmchilton/blend4j/galaxy/JobsClient.java

```diff
@@ -0,0 +1,32 @@
+package com.github.jmchilton.blend4j.galaxy;
+
+import java.util.List;
+
+import com.github.jmchilton.blend4j.galaxy.beans.Job;
+import com.github.jmchilton.blend4j.galaxy.beans.JobDetails;
+
+/**
+ * Client for interacting with Galaxy's Job API.
+ * 
+ * @author Franklin Bristow franklin.bristow@phac-aspc.gc.ca
+ *
+ */
+public interface JobsClient {
+
+	/**
+	 * Get *all* jobs executed by Galaxy. **WARNING**: This may take a long time
+	 * to execute, depending on how many jobs Galaxy has executed in its
+	 * history.
+	 * 
+	 * @return a collection of all jobs executed by Galaxy.
+	 */
+	public List<Job> getJobs();
```

Comment:
So, bioblend has the ability to pass in extra parameters to limit the amount of results that are returned.  In particular, you can pass a `history_id` to limit jobs to a specific history https://bioblend.readthedocs.org/en/latest/api_docs/galaxy/all.html#module-bioblend.galaxy.jobs and https://github.com/galaxyproject/bioblend/blob/master/bioblend/galaxy/jobs/__init__.py.

I assume we're really only interested in Jobs from a specific history so could you add the ability to limit results like this?  If we return all results my assumption is that we're quickly going to run into issues with lots and lots of data returned by Galaxy (which will constantly grow).